play/src/front/Components/ActionBar/ActionBar.svelte

```diff
@@ -574,24 +586,48 @@
                                     {/if}
                                 </button>
 
-                                <!-- microphone list -->
                                 <div
                                     class={`wa-dropdown-menu ${microphoneActive ? "" : "tw-invisible"}`}
                                     style="bottom: 15px;right: 0;"
                                     on:mouseleave={() => (microphoneActive = false)}
                                 >
-                                    {#each $microphoneListStore as microphone}
-                                        <span
-                                            class="wa-dropdown-item"
-                                            on:click|stopPropagation|preventDefault={() =>
-                                                selectMicrophone(microphone.deviceId)}
+                                    {#if $microphoneListStore.length > 0}
+                                        <!-- microphone list -->
+                                        <span class="tw-underline tw-font-bold tw-text-xs tw-p-1"
+                                            >{$LL.actionbar.subtitle.microphone()} 🎙️</span
                                         >
-                                            {StringUtils.normalizeDeviceName(microphone.label)}
-                                            {#if selectedMicrophone === microphone.deviceId}
-                                                <CheckIcon size="13" />
-                                            {/if}
-                                        </span>
-                                    {/each}
+                                        {#each $microphoneListStore as microphone}
+                                            <span
+                                                class="wa-dropdown-item"
+                                                on:click|stopPropagation|preventDefault={() =>
+                                                    selectMicrophone(microphone.deviceId)}
+                                            >
+                                                {StringUtils.normalizeDeviceName(microphone.label)}
+                                                {#if selectedMicrophone === microphone.deviceId}
+                                                    <CheckIcon size="13" />
+                                                {/if}
+                                            </span>
+                                        {/each}
+                                    {/if}
+
+                                    <!-- speaker list -->
```

Comment:
Slight change: I would probably put the speaker list at the top and the microphone list at the bottom. Because when you click the microphone icon, you most probably want to change the microphone and your mouse cursor is at the bottom.